Various documentation improvements (#919)
tradingview/lightweight-charts: website/docs/intro.md

The first thing you need to do to use `lightweight-charts` is to install it from [npm](https://www.npmjs.com/):

```console
npm install --save lightweight-charts
```

_Note that the package is shipped with TypeScript declarations, so you can easily use it within TypeScript code._

## Creating a first chart

> Since the library has been installed in your repo you're ready to create your first chart.

Reviewer: ```suggestion
Once the library has been installed in your repo you're ready to create your first chart.
```

> First of all, you need to create a source file where you will create a chart if you don't have it yet.
> Then in this file you need to import the library:

Author: ```suggestion
First of all, in a file where you would like to create a chart you need to import the library:
```

@edew something like that?

Reviewer: 👍

```js
import { createChart } from 'lightweight-charts';
```

> [`createChart`](/api#createchart) is an entry-point for creating charts and you can use it as many times as many charts you need to create:

Reviewer: ```suggestion
[`createChart`](/api#createchart) is the entry-point for creating charts. You can use it to create as many charts as you need:
```

## Creating a series

Since your chart is created it is ready to display a data.

The basic primitive to display a data is [a series](/api/interfaces/ISeriesApi).
It could be one of possible series types:

- Area
- Bar
- Baseline
- Candlestick
- Histogram
- Line

To create a series with desired type you need to use appropriate method from [`IChartApi`](/api/interfaces/IChartApi).
All of them have the same naming `add<type>Series`, where `<type>` is a type of a series you'd like to create:

```js
import { createChart } from 'lightweight-charts';

const chart = createChart(container);

const areaSeries = chart.addAreaSeries();
const barSeries = chart.addBarSeries();
const baselineSeries = chart.addBaselineSeries();
// ... and so on
```

Please look at [this page](/series-types.md) for more information about different series types.

Note that **a series cannot be transferred from one type to another one** since different series types have different data and options types.

## Setting and updating a data

Once your chart and series are created it's time to set a data to the series.

> Note that regardless of a series type, the API calls are the same for all series types (the type of the data might be different though).

Reviewer: ```suggestion
Note that regardless of the series type, the API calls are the same (the type of the data might be different though).
```
